ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -514,180 +520,21 @@ ${SHARED_VPC_SWITCH} \
 ${SECURITY_GROUP_ID_SWITCH} \
 ${NO_CNI_SWITCH} \
 ${DRY_RUN_SWITCH}
-" | sed -E 's/\s{2,}/ /g' > "${SHARED_DIR}/create_cluster.sh"
+" 
+echo "$cmd"| sed -E 's/\s{2,}/ /g' > "${SHARED_DIR}/create_cluster.sh"
 cat "${SHARED_DIR}/create_cluster.sh" | sed "s/$AWS_ACCOUNT_ID/$AWS_ACCOUNT_ID_MASK/g" > "${ARTIFACT_DIR}/create_cluster.sh"
```

Comment:
it's a little confused between `${SHARED_DIR}/create_cluster.sh` and `${ARTIFACT_DIR}/create_cluster.sh`, suggest to 
```suggestion
cat "${SHARED_DIR}/create_cluster.sh" | sed "s/$AWS_ACCOUNT_ID/$AWS_ACCOUNT_ID_MASK/g" > "${ARTIFACT_DIR}/create_cluster_command_for_debugging.txt"
```
or something  like that?

Reply:
Seems like we do not need `${ARTIFACT_DIR}/create_cluster.sh` for we have print it in the terminal.